cirq-core/cirq/devices/noise_utils.py

```diff
@@ -97,8 +96,10 @@ def _from_json_dict_(cls, gate_type, qubits, **kwargs) -> 'OpIdentifier':
         return cls(gate_type, *qubits)
 
 
-# TODO: expose all from top-level cirq?
-def decay_constant_to_xeb_fidelity(decay_constant: float, num_qubits: int = 2) -> float:
+@deprecated(deadline='v2.0', fix='use qis.decay_constant_to_xeb_fidelity')
```

Comment:
Nit: let's suggest full import name here and below

```suggestion
@deprecated(deadline='v2.0', fix='use cirq.qis.decay_constant_to_xeb_fidelity')
```

Reply:
done